Improve support feedback tab

mzayeddfe opened the pull request:
# Brief overview of changes

- Added extra arguments to support_panel()
- Created html_paragraph_tags() 
- Changed a test in test-support-panel (line 175)

## Why are these changes being made?

Users wanted more flexibility on how the support panel looks. 

## Detailed description of changes

Details for extra arguments in support_panel():

- feedback_custom_text - add custom text under the feedback heading
- info_custom_text - add custom text under the find out more information heading
- contact_custom_text - add custom text under the contact us heading
- extra_text - add extra sections before the contact us section

html_paragraph_tags() supports with the extra arguments above by generating tags using shiny::tagList() and it generated a heading(optional) and a body (paragraph). 

Changed a test in test-support-panel (line 175). This is because I put another shiny::tags$div() around the if statement for the feedback section if there no extra text. This changes the position of the title in the output for the test on that line even though the outputs are as expected. 

## Issue ticket number/s and link

[54](https://github.com/dfe-analytical-services/dfeshiny/issues/54)

cjrace commented on R/support_panel.R:
```diff
@@ -14,9 +14,21 @@
+#' @param info_custom_text A single vector or a combined vector wrapped in
+#' `shiny::tagList()` instead of c() for custom text to go under the
+#' "Find out more information on the data" heading.
+#' @param contact_custom_text A single vector or a combined vector wrapped in
```
I'm not sure we'd want to allow users to edit the contact bit too much, so I'd be tempted to drop this argument, don't think there was a specific ask for this in #54 and having a consistent approach to contact details is probably better for users

cjrace commented on R/support_panel.R:
```diff
@@ -14,9 +14,21 @@
+#' @param feedback_custom_text A single vector or a combined vector wrapped in
+#' `shiny::tagList()` instead of c() for custom text to go under the "Give us
+#' feedback" heading.
+#' @param info_custom_text A single vector or a combined vector wrapped in
```
Would `custom_data_info` be a slightly more descriptive name for this?

cjrace commented on R/support_panel.R:
```diff
@@ -14,9 +14,21 @@
 #' @param alt_href Alternative link to the parent publication (if not hosted on
 #' Explore Education Statistics)
 #' @param form_url URL for a feedback form for the dashboard
+#' @param feedback_custom_text A single vector or a combined vector wrapped in
```
Not sure we need to add the option to customise the feedback section? Can't see an ask for it in #54?

mzayeddfe commented on R/support_panel.R:
```diff
@@ -45,14 +57,110 @@
+#' # Adding custom text to the feedback section only
+#'
+#' support_panel(
+#'   team_email = "my.team@@education.gov.uk",
```
emails appear normally on documentation and the extra @ is there for escaping the first @. cam happy for this comment to be resolved.

mzayeddfe commented:
@cjrace 
I added h_level as an arg but added it so it’s after body in the section_tags() function. This is so lazy inputs of section_tags(“heading”, “body”) still work without specifying what each quote is for but happy to amend if you have different thoughts. 

I also added charlotte’s changes about bad_link to the news but let me know if you want me to change anything there :)

cjrace reviewed (changes requested):
Couple of minor things to tidy up then we'll be good to go!
